pkg/cmd/server/start/node_args.go

```diff
@@ -138,12 +138,7 @@ func (args NodeArgs) BuildSerializeableNodeConfig() (*configapi.NodeConfig, erro
 		config.ServingInfo.ClientCA = admin.DefaultKubeletClientCAFile(args.MasterCertDir)
 	}
 
-	// Roundtrip the config to v1 and back to ensure proper defaults are set.
```

Comment:
We don't support it on the kubelet, it's not required for this commit.  This fix only requires DNS and the master to get DNS back to sanity.